.travis.yml

```diff
@@ -1,5 +1,14 @@
+sudo: false
 language: node_js
 node_js:
   - 'node'
+before_install:
+  - /bin/sh ./scripts/install-atom.sh
+install:
+  - npm install
+  - npm rebuild
+script:
+  - npm test
+  - /bin/sh ./scripts/test-atom.sh
```

Comment:
Would cause the integration test to not break the build if failing: https://docs.travis-ci.com/user/customizing-the-build#Breaking-the-Build